sdk/storage/CONTRIBUTING.md

```diff
@@ -18,28 +18,38 @@ The Azure Storage development team uses Visual Studio Code. However, any preferr
 
 ### Configuration
 
-The only step to configure testing is to set the appropriate environment variables. Create environment variables named "ACCOUNT_NAME", "ACCOUNT_KEY" or "ACCOUNT_SAS". The first two will be used for most requests. The "ACCOUNT_SAS" will only be used for tests in browsers.
+The only step to configure testing is to set the appropriate environment variables. Create environment variables named
+
+```bash
+ACCOUNT_NAME
```

Comment:
For dfs, the env var ACCOUNT_* are with DFS prefix. Table suggested by Ramya is a good approach to differ env var usage in different packages.

Reply:
Since the DFS* ones are only for storage-file-data-lake package, it makes more sense to mention them in that package readme or test/readme.

Reply:
Will be taken care here https://github.com/Azure/azure-sdk-for-js/issues/6830